operator-common/src/main/java/io/strimzi/operator/common/Util.java

```diff
@@ -369,4 +371,24 @@ public static ConfigResource getBrokersConfig(int podId) {
     public static ConfigResource getBrokersConfig(String podId) {
         return new ConfigResource(ConfigResource.Type.BROKER, podId);
     }
+
+    public static String getStringHash(String toBeHashed)  {
```

Comment:
This should be just called `stringHash()`. Why the fetish with prefixing every method which returns something with `get`?

Also some Java doc would not go amiss.